.travis.yml

```diff
@@ -44,24 +52,26 @@ install:
   - popd 
 
 before_script:
-  - sudo minikube start --vm-driver=none --cpus=1 --memory=1024
-  - sudo minikube update-context
-  # By default root is the owner of the .minikube directory. This is changed to $USER so that
-  # from the integration tests which run as a non-root user can access the files and connect
-  # to the minikube cluster
-  - sudo chown -R $USER $HOME/.minikube
-  - sudo chown -R $USER $HOME/.kube
+  - if [ "$XC_ARCH" == "amd64" ]; then
+      sudo minikube start --vm-driver=none --cpus=1 --memory=1024;
```

Comment:
the `--cpus` and `--memory` flag can be removed. If VM driver is none, these flags are not considered.